### PR TITLE
Index page sponsors

### PR DESCRIPTION
**Pull request by uagarwal744**

### CODE REVIEW

**Comment by uagarwal744** [comment]
@icyflame Forgot password also added in this PR

File: acco-travel-form-continue.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 $cabWhere=$_POST['cabWhere'];
 $cabWhen=$_POST['cabWhen'];
 $cabDate=$_POST['cabDate'];
+$cabPpl=$_POST['cabPpl'];		//To be added in database column
```
Comment (icyflame) [inline acco-travel-form-continue.php:19]: @uagarwal744 Data type?

**Comment by uagarwal744** [comment]
@icyflame int will do, it is the number of people travelling in cab

File: sponsors2016.php [hunk 2]
```diff
@@ -125,7 +125,11 @@
 	<h2 style="text-align:center">Fine Arts Photography</h2>
 	<div class="row">
 	<div class=""><a href="" target="_blank" >
+<<<<<<< HEAD
```
Comment (icyflame) [inline sponsors2016.php:128]: I am fixing the conflict this time. From next time, make sure that conflicts don't make it into Pull Requests.

**Pull request merged**